.azure/azure-pipelines.perf.yml

```diff
@@ -18,6 +18,7 @@ parameters:
   - PGO
   - Record
   - RecordQUICTrace
+  - Kernel
```

Comment:
I don't want to have to pick between kernel and everything else. Ideally it should either always be included to a separate parameter.